docs/guide/device-and-platform-support.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Device & Platform Support"
+type: guide
+layout: docs
+parent_section: guide
+order: 5
+---
+
+Devices and platform support depends on how well the browsers support certain
+devices and APIs. A-Frame supports both flat (3D on a normal screen) and WebVR
+experiences.  Though its focus is heavily VR. We will break up support into two
+categories: flat and VR.
+
+<!--toc-->
+
+## Support for Flat Experiences
+
+Support for flat experiences primarily depends on the browsers' WebGL support.
+We can see what browsers support WebGL at [Can I use WebGL?][caniusewebgl].
+
+## Support for VR Experiences
+
+Support for VR experiences, along with WebGL support, depends on the browsers'
+WebVR API support. The browsers that support the WebVR API *at the moment you
```

Comment:
not sure why we need to say `*at the moment you read this*` if you're not specifically calling out any browsers